snippets/insertIntoBST.md

```diff
@@ -0,0 +1,24 @@
+### insertIntoBST
+
+Inserts a new node into binary tree.
+
+Creates and inserts new node into existing tree With the help of `createBST` function.
+
+```js
+const createBST = value => ({value: value, right: null, left: null});
```

Comment:
This feels like it's something we said we wanted to avoid in our own contribution guidelines isn't it?